[Rules] New create DNS record pop-up (#20276)
cloudflare/cloudflare-docs: src/content/docs/rules/reference/troubleshooting.mdx

Cloudflare may remove the `Content-Length` header from responses delivered to website visitors. If the visitor must receive the `Content-Length` header, configure the origin server to include a `cache-control: no-transform` HTTP header in the response.

## This rule may not apply to your traffic

> If your rule expression is matching a hostname for which you have not created a DNS record or are not proxying its traffic through Cloudflare, you will get a pop-up window with a couple of options:

Reviewer: ```suggestion
If your rule expression is matching a hostname for which you have neither created a DNS record nor enabled proxying traffic through Cloudflare, you will get a pop-up window with a couple of options:
```
